### PR TITLE
Add models for Jaco box pushing

### PR DESCRIPTION
**Pull request by aykut-tri**
* Renames the box to push model by a more specific name: Intel NUC box
* Adds a new 15-cm box model that is a better fit to Jaco
* Adds the Jaco versions of the iiwa box pushing environments

### CODE REVIEW

File: models/box_to_flip.sdf [hunk 1]
```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0"?>
```
Comment (JoseBarreiros-TRI) [inline models/box_to_flip.sdf:1]: how is this model different than box_intel_nuc.sdf?  also, by naming conventions used in other files, should this have an "intel_nuc" in the name?

File: models/wall_box.sdf [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <pose> 0 0 0 0 0 0</pose>
```
Comment (JoseBarreiros-TRI) [inline models/wall_box.sdf:5]: nit: could you rename this as wall_lenght_widht_depth.sdf

**Review by JoseBarreiros-TRI — COMMENTED** [review]
just a few nits for naming the models.

File: models/wall_box.sdf (hunk 2 above)
Comment (aykut-tri) [reply models/wall_box.sdf:5]: Done, thanks! Also, I reverted the original wall file, as it is used by other examples.

File: models/box_to_flip.sdf (hunk 1 above)
Comment (aykut-tri) [reply models/box_to_flip.sdf:1]: Good point, I renamed it as such. The differences are the additional corner collisions in the top surface and the increased friction coefficients.

**Pull request merged**